packages/es-dev-server/test/middleware/compatibility-transform.test.js

```diff
@@ -19,7 +19,7 @@ async function setupServer(compatibility) {
 }
 
 async function fetchText(url, userAgent) {
-  const response = await fetch(`${host}${url}`, {
+  const response = await fetch(`${host}${url}?transform-modules`, {
```

Comment:
this enabled the transform for each test, right? probably best to do this only for specific systemjs tests

Reply:
Yep, it did. I had quickly looked at a couple of these files, and it seemed they used modules. But only two do apparently. 🙈  Will change!